tools/update_compat_level.py

```diff
@@ -0,0 +1,14 @@
+"""Updates the compat level by running shell command"""
+
+import os
+import sys
+
+def update_compat_level():
+    """Updates the compat level to the value provided in argument"""
+    compat_level = sys.argv[1]
+    command_to_execute = "for d in DEB/*/ ; do (cd ""$d"" && echo "+ \
+        compat_level+" > debian/compat); done"
```

Comment:
do you also need to create debian/control? [this](https://manpages.debian.org/testing/debhelper/debhelper.7.en.html#COMPATIBILITY_LEVELS) man page states


```
Prior versions of debhelper required specifying the compatibility level in the file debian/compat, and current debhelper still supports this for backward compatibility. To use this method, the debian/compat file should contain the compatibility level as a single number, and no other content. If you specify the compatibility level by this method, your package will also need a versioned build dependency on a version of the debhelper package equal to (or greater than) the compatibility level your package uses. So, if you specify compatibility level 13 in debian/compat, ensure debian/control has:

  Build-Depends: debhelper (>= 13~)
```

Reply:
debian/control file is already present in there.

Reply:
Yes, the file is there, but it doesn't have the required `Build-Depends` entry. It sounds like `compat` is deprecated, and `control` is the new, preferred way to manage this. That's why it says
> ensure debian/control has:
> Build-Depends: debhelper (>= 13~)

It may work like this right now, but it might also stop working without warning...

Reply:
So when we are updating the compat level should we also update the control file with this value?